pom.xml

```diff
@@ -37,6 +37,7 @@
   <version>1.0.1-SNAPSHOT</version>
   <packaging>pom</packaging>
   <name>Declarative Pipeline Migration Assistant</name>
+  <description>A plugin which converts existing Freestyle project into a starter Declarative Jenkinsfile</description>
```

Comment:
This is the parent, not the converter plugin, so I'd move this content to `declarative-pipeline-migration-assistant/pom.xml`

Reply:
I ended up making it match what's in [index.jelly](https://github.com/jenkinsci/declarative-pipeline-migration-assistant-plugin/blob/master/declarative-pipeline-migration-assistant/src/main/resources/index.jelly#L3). Can make them both have the longer description, if that's better.